cypress/integration/user-claimer/2-view-and-claim-w-game.js

```diff
@@ -0,0 +1,240 @@
+				
+				// Expect response after human validation
+				cy.contains( 'Prepping your POAP' )
+				
+				// Human game welcome screen 
+				cy.contains( 'Play a game' )
+
+				// Click start game button
+				cy.contains( 'button', 'Start game' ).click()
+
+				// Expect score text
+				cy.contains( 'Score: 0 of' )
+
+				// Find buttons in page and click the one with h1 span value
+				cy.get( 'h1 span' ).invoke( 'text' ).then( (text) => { cy.contains( 'button', text ).click( {force: true} ) })
+				cy.wait(1000)
+
+				// Expect score text
+				cy.contains( 'Score: 1 of' )
+
+				// Find buttons in page and click the one with h1 span value
+				cy.get( 'h1 span' ).invoke( 'text' ).then( (text) => { cy.contains( 'button', text ).click( {force: true} ) })
+				cy.wait(1000)
+
+				// Expect score text
+				cy.contains( 'Score: 2 of' )
+
+				// Expect winning screen
+				cy.contains( 'You won!' )
+
+				// Click retrieval link
+				cy.contains( 'button', 'Claim your' ).click()
+
+				// Wait for code retrieval
+				cy.contains( 'POAP Claim' )
+
+				// Check if POAP link supplies one of the test codes
+				cy.get( 'input' ).invoke( 'val' ).then( val => {
+					
+					expect( val ).to.be.oneOf( oneCode )
+
+				} )
+
+			} )
+		
+
+	} )
+
+	it( 'Event 1: Shows code marked as used (previous redirect marked as used)', function( ) {
+
+		// Visit the public link
+		cy.visit( this.event_1_publiclink )
+
+		// Accept disclaimer
+		cy.get( '#event-view-accept-disclaimer' ).click()
+
+		// Shows one code as claimed
+		cy.contains( '1 of 1 codes' )
+
+	} )
+
+	it( 'Event 1: Previous challenge link no longer works', function( ) {
+
+		// Visit the public link
+		cy.visit( this.event_1_first_challenge_url )
+
+		// Interface should indicate that the link expired
+		cy.contains( 'This link was already used' )
+
+	} )
+
+	it( 'Event 1: Shows no codes after code is scanned', function( ) {
+
+		// Visit public event link
+		cy.visit( this.event_1_publiclink )
+
+		// Accept disclaimer
+		cy.get( '#event-view-accept-disclaimer' ).click()
+
+		cy.contains( '1 of 1 codes' )
+
+	} )
+
+	it( 'Event 1: Shows error if link was used after code ran out', function( ) {
+
+		// Visit the public link to the second code as read by simulating a scan
+		cy.request( { ...request_options, url: `${ Cypress.env( 'REACT_APP_publicUrl' ) }/claim/${ this.event_1_public_auth_link }` } ).as( `request` )
+			.then( extract_challenge_from_url )
+			.then( event_1_second_challenge => {
+
+				// Visit the challenge link
+				cy.visit( `/claim/${ event_1_second_challenge }` )
+
+				// Human game welcome screen 
+				cy.contains( 'Play a game' )
+
+				// Click start game button
+				cy.contains( 'button', 'Start game' ).click()
+
+				// Expect score text
+				cy.contains( 'Score: 0 of' )
+
+				// Find buttons in page and click the one with h1 span value
+				cy.get( 'h1 span' ).invoke( 'text' ).then( (text) => { cy.contains( 'button', text ).click( {force: true} ) })
+				cy.wait(1000)
```

Comment:
These waits were optical right? If you no longer need them please remove to the CI can run as fast as it is able.

Reply:
The wait is not really a hold up as the game does need to run for a minimum of 10 sec. Removed it anyway.